### PR TITLE
Simulating pinch gesture to get zoom.

### PR DESCRIPTION
**Pull request by juliocri**
This branch is an initial sample of pinch gesture over the board.

-- tested with real device (galaxy s6).

### CODE REVIEW

File: board-viewer.html [hunk 1]
```diff
@@ -180,10 +179,10 @@
 Polymer({
   is: "board-viewer",
   properties: {
-    /** Whether the zoom control is visible. */
+    /** Whether the zoom operations are available / control is visible. */
     disableZoom: {
       type: Boolean,
-      value: true,
+      value: false,
```
Comment (juliocri) [inline board-viewer.html:188]: What do you think to invert the default value of this variable?

File: board-viewer.html [hunk 2]
```diff
@@ -270,10 +278,67 @@
...
+    "touchmove": "_onPinch",
+    "wheel": "_onScroll",
     "board.response": "_boardLoaded",
     "board.error": "_boardLoadError"
   },
 
+  _onScroll: function(event) {
+    /* Inverted aim. */
+    if (event.deltaY < 0) {
+      this.zoom = Math.max(Math.min(this.zoom + 0.1, 2.0), 0);
```
Comment (juliocri) [inline board-viewer.html]: I did some test with this values most part of the time and I see a good operability, min-zoom as 0.5 does not look so well, I think the zoom should to return to original size (0), 2.0 is not so big, not so small, this is only my personal perception.

File: board-viewer.html (hunk 1 above)
Comment (jketreno) [reply board-viewer.html:188]: Outside of interactive explorer apps (like the main view we have built,) the typical embedding will be as a clipped and zoomed region used for static (non interactive) illustration and call-out documentation, so the default should be to have the interactive elements (zoom, drag) disabled.

Eventually when zoom is enabled, there will be a corresponding on screen zoom bar as well.

**Comment by juliocri** [comment]
@jketreno just added a sample of a zoom bar, not so bad for default styling.

**Pull request merged**